content/en/setup/module-1/core-concepts.mdx

```diff
@@ -68,10 +68,10 @@ A DDO document is composed of standard DDO attributes:
 
 ### DDO Services
 
-Each type of asset (dataset, algorithm, etc, ..) typically will have associated different kind of services. This are the types of services that are commonly added to all the assets:
+Each type of asset (dataset, algorithm, etc, ..) typically will have associated different kinds of services. These are the types of services that are commonly added to all the assets:
 
 - metadata - describing the asset
-- access - describe conditions to be fulfilled in order to consume this data
+- access - describe conditions to be fulfilled in order to download this data.
```

Comment:
Access isn't just about downloading the data, it can also be running a compute job. Maybe it would be better to say:

"in order to download or run an algorithm on the data."